presto-spi/src/main/java/com/facebook/presto/spi/constraints/TableConstraintsHolder.java

```diff
@@ -29,10 +33,20 @@ public class TableConstraintsHolder
     private final List<TableConstraint<String>> tableConstraints;
     private final Map<String, ColumnHandle> columnNameToHandleAssignments;
 
+    public static void validateTableConstraints(Collection<TableConstraint<String>> constraints)
```

Comment:
I would move this below the constructor because 
1. we usually have constructors first 
2. we put method definitions below where they get called.